pkgs/by-name/ks/kseexpr/package.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, lib
+, fetchFromGitLab
+, sd
+, cmake
+, qt5
+, libsForQt5
+, bison, flex, llvm, extra-cmake-modules
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "kseexpr";
+  version = "4.0.4.0";
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "graphics";
+    repo = pname;
```

```suggestion
    repo = "kseexpr";
```
semantically different